app/src/main/java/com/expidev/gcmapp/support/v4/fragment/SettingsFragment.java

```diff
@@ -169,6 +187,16 @@ private void updateMccPrefSummary() {
         }
     }
 
+    private class MinistryChangeListener implements Preference.OnPreferenceChangeListener {
+        @Override
+        public boolean onPreferenceChange(Preference preference, Object newValue) {
+            // update summary as necessary
+            updateMinistryPrefSummary();
```

Comment:
I need to return true.

`@return True to update the state of the Preference with the new value.`